src/libraries/System.Private.CoreLib/src/System/Decimal.cs

```diff
@@ -243,10 +243,17 @@ public static long ToOACurrency(decimal value)
         // The possible binary representations of a particular value are all
         // equally valid, and all are numerically equivalent.
         //
-        public Decimal(int[] bits)
+        public Decimal(int[] bits) :
+            this(bits != null ? bits.AsSpan() : throw new ArgumentNullException(nameof(bits)))
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="decimal"/> to a decimal value represented in binary and contained in the specified span.
+        /// </summary>
+        /// <param name="bits">A span of four <see cref="int"/>s containing a binary representation of a decimal value.</param>
```

Comment:
Add the exception tag for invalid bits and for bits.Length != 4.

https://docs.microsoft.com/en-us/dotnet/api/system.decimal.-ctor?view=netframework-4.8#System_Decimal__ctor_System_Int32___

Reply:
Done